src/mixins/filing-mixin.ts

```diff
@@ -25,7 +25,7 @@ export default class FilingMixin extends Vue {
   /**
    * Flattens and sorts an array of comments.
    * @param comments the array of comments to sort and deconstruct
-   * @return the sorted and flattened array of comments
+   * @returns the sorted and flattened array of comments
```

Comment:
I've seen this! What is the difference?

Reply:
Both are allowed, but when I hover over the function I prefer to see "returns XXX" instead of "return XXX" :)